lib/sqitch-check.pod

```diff
@@ -0,0 +1,113 @@
+=head1 Name
+
+sqitch-check - Runs checks
```

Comment:
I'd make this a little more verbose, something like `Checks for divergences between planned and deployed changes`.